sklearn/multioutput.py

```diff
@@ -468,6 +476,9 @@ def fit(self, X, Y, **fit_params):
         self : object
         """
         X, Y = self._validate_data(X, Y, multi_output=True, accept_sparse=True)
+        if len(Y.shape) != 2:
+            raise ValueError("invalid Y for multi-label fit. "
+                             "Y must be of shape (n_samples, n_classes)")
```

Comment:
I' m confused is that related to the verbose issue in some way?

Reply:
I don't think it is. @efiegel Could you revert this change

Reply:
It's not related to verbose, no. I reverted here and will add another PR for it.